react-validated-input-component/validated-input-app/src/ValidatedInput.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+import { HiX } from 'react-icons/hi';
+import { HiCheck } from 'react-icons/hi';
+
+export default function ValidatedInput() {
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('A password is required');
+  const [validation, setValidation] = useState(false);
+
+  function handlePassword(event) {
+    console.log('password: ', { password });
```

Comment:
Great job using a console log here for verifying what the value in `password` is, but remember to cleanup console logs before submitting pull requests, we typically don't want console logs in our production code.